src/App.jsx

```diff
@@ -15,14 +27,24 @@ function App() {
   };
 
   return (
-    <ChakraProvider>
-      <DropZone setFiles={setFiles} />
-      <Button onClick={onSubmit}>Upload File</Button>
-      {images.map(e => (
-        <Image key={e} src={e} />
-      ))}
-      <DonationForm />
-    </ChakraProvider>
+    <Router>
+      <ChakraProvider>
+        <DropZone setFiles={setFiles} />
+        <Button onClick={onSubmit}>Upload File</Button>
+        {images.map(e => (
+          <Image key={e} src={e} />
+        ))}
+        <DonationForm />
+      </ChakraProvider>
+      <Routes>
```

Comment:
put routes inside chakra provider in order to give all of the pages the same chakra styling